client/galaxy/scripts/components/Toolshed/Categories.vue

```diff
@@ -65,7 +65,7 @@ export default {
                 });
         },
         onCategory(category) {
-            this.$emit("onCategory", category);
+            this.$emit("onCategory", `category:${category}`);
```

Comment:
You need to enclose the category in single quotes `'` for categories that contain e.g. space

Reply:
`_parse_reserved_filters("climate category:'Climate Analysis' owner:'bjoern gruening' psy_maps")`

Reply:
Cool, Ok.